docs/openfaas-cloud/self-hosted/index.md

```diff
@@ -0,0 +1,5 @@
+# Self-hosted
```

Comment:
Is this a new file? How does it affect existing links to these pages?

Reply:
It's a new file yes linked from other one:
see  +* [OpenFaaS Cloud: Self-hosted](/openfaas-cloud/self-hosted/)

Reply:
Please could you update the link without adding a new page? 🙏

Thank you!